source/frontend/src/features/leases/detail/LeasePages/details/UpdateLeaseForm.tsx

```diff
@@ -37,11 +37,11 @@ export const UpdateLeaseForm: React.FunctionComponent<IUpdateLeaseFormProps> = (
         variant: 'warning',
         okButtonText: 'Yes',
         cancelButtonText: 'No',
-        message: `You've marked this status file as a duplicate. If you save it, you'll still see it in the management table.
+        message: `Selecting the Duplicate file status will hide the file within the Property Information, PIMS Files tab. The file will remain searchable in the Manage Lease/Licence Files advanced search.
 
-        Please ensure that all related documents and notes are moved to the main file.
+        Please ensure that all related documents and notes are moved to the active file.
 
-        Do you want to acknowledge and proceed?`,
+        Do you want to proceed?`,
         title: 'Warning',
         handleCancel: () => {
```

Comment:
According to JIRA the popup title should read "Acknowledgement" - please change from Warning
![image](https://github.com/user-attachments/assets/3dcc7146-7571-4280-a310-b83342a7c3ac)